src/test/resources/output/json/00594/HT005944358.json

```diff
@@ -0,0 +1,105 @@
+    },
+    "heldBy" : {
+      "id" : "http://lobid.org/organisations/DE-361",
+      "label" : "lobid Organisation"
+    },
+    "id" : "http://lobid.org/items/HT005944358:DE-361:NA000%20Z540%5B95#!",
+    "itemOf" : "http://lobid.org/resources/HT005944358#!",
+    "label" : "lobid Bestandsressource",
+    "note" : "00000015",
+    "type" : [ "Item" ]
+  } ],
+  "hbzId" : "HT005944358",
+  "id" : "http://lobid.org/resources/HT005944358#!",
+  "isPartOf" : [ {
+    "id" : "http://ld.zdb-services.de/resource/200075-1",
+    "label" : "http://ld.zdb-services.de/resource"
+  }, {
+    "id" : "http://lobid.org/resources/HT005077038#!",
+    "label" : "lobid Ressource"
+  } ],
+  "language" : [ {
+    "id" : "http://id.loc.gov/vocabulary/iso639-2/deu",
+    "label" : "Deutsch"
+  } ],
+  "medium" : [ {
+    "id" : "http://purl.org/lobid/lv#Miscellaneous",
+    "label" : "Miscellaneous"
+  } ],
+  "publication" : [ {
+    "location" : [ "Berlin [u.a.]", "Halle", "Tübingen" ],
+    "publishedBy" : [ "Neomarius", "de Gruyter", "Niemeyer" ],
+    "type" : [ "PublicationEvent" ]
+  } ],
+  "sameAs" : [ {
+    "id" : "http://hub.culturegraph.org/resource/HBZ-HT005944358",
+    "label" : "Culturegraph Ressource"
+  } ],
+  "subject" : [ {
+    "id" : "http://dewey.info/class/850/",
+    "label" : "Italienische, rumänische, rätoromanische Literaturen"
+  }, {
+    "id" : "http://d-nb.info/gnd/4050484-0",
+    "label" : "Romanistik",
+    "type" : [ "SubjectHeading" ]
+  }, {
+    "id" : "http://dewey.info/class/840/",
+    "label" : "Literaturen romanischer Sprachen; Französische Literatur"
+  }, {
+    "id" : "http://d-nb.info/gnd/4115788-6",
+    "label" : "Romanische Sprachen",
+    "type" : [ "SubjectHeading" ]
+  }, {
+    "id" : "http://d-nb.info/gnd/4035964-5",
+    "label" : "Literatur",
+    "type" : [ "SubjectHeading" ]
+  }, {
+    "label" : "Zeitschrift",
+    "type" : [ "SubjectHeading" ]
+  }, {
+    "id" : "http://dewey.info/class/440/",
+    "label" : "Romanische Sprachen; Französisch"
+  }, {
+    "id" : "http://dewey.info/class/860/",
+    "label" : "Spanische, portugiesische Literaturen"
+  } ],
+  "subjectAltLabel" : [ "Romanische Philologie", "Schöne Literatur", "Dichtung", "Sprachkunst", "Romanisch (Sprachfamilie)", "Wortkunst", "Belletristik" ],
+  "subjectChain" : [ "Romanische Sprachen | Literatur | Zeitschrift", "Romanistik | Zeitschrift", "Romanische Sprachen | Zeitschrift" ],
+  "subjectOrder" : [ "http://d-nb.info/gnd/4115788-6, Zeitschrift", "http://d-nb.info/gnd/4115788-6, http://d-nb.info/gnd/4035964-5, Zeitschrift", "http://d-nb.info/gnd/4050484-0, Zeitschrift" ],
+  "title" : "Zeitschrift für romanische Philologie, 95",
+  "type" : [ "BibliographicResource", "Miscellaneous" ],
+  "volumeIn" : [ {
+    "multiVolumeWork" : [ {
+      "id" : "http://lobid.org/resources/HT005077038#!",
+      "label" : "lobid Ressource"
+    } ],
+    "numbering" : "95",
+    "type" : [ "MultiVolumeWorkRelation" ]
+  } ]
+}
```

Comment:
Is this accidentally added ? ~~This resource has item informations that are not yet part of it's the parent resource.~~